app/src/main/java/fr/free/nrw/commons/upload/depicts/handleDepictsDoa.kt

```diff
@@ -0,0 +1,63 @@
+package fr.free.nrw.commons.upload.depicts
```

Comment:
This is not needed I guess, Room handles the threading part on its own, even if it does not why make a separate class and not let the callers switch threads as needed?